mmv1/products/datacatalog/Taxonomy.yaml

```diff
@@ -23,6 +23,7 @@ references: !ruby/object:Api::Resource::ReferenceLinks
   guides:
     'Official Documentation': https://cloud.google.com/data-catalog/docs
   api: https://cloud.google.com/data-catalog/docs/reference/rest/v1/projects.locations.taxonomies
+# Skip sweeper as a handwritten sweeper is implemented
```

Comment:
Can you state the specific issue with the generated sweeper? i.e. `Skip sweeper due to a pre-delete step that was required. Implemented as a handwritten sweeper in {{PR}}`

Reply:
Comment updated.